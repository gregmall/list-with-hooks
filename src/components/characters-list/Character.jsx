import React from 'react';
import PropTypes from 'prop-types';

const Character = ({ image, name }) => (
  <div>
    <p>{name}</p>
    <img src={image}/>
    
    </div>
  
 );

Character.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
  
};

export default Character;
