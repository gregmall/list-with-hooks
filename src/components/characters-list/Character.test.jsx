import React from 'react';
import { render } from '@testing-library/react';
import Character from './Character';

describe('Character component', () => {
  it('renders Character component', () => {
    const { asFragment } = render(<Character name="rick" image="www.url.com" />);

    expect(asFragment()).toMatchSnapshot();
  });
});

