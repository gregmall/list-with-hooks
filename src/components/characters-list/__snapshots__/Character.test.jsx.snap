// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character component 1`] = `
<DocumentFragment>
  <div>
    <p>
      rick
    </p>
    <img
      src="www.url.com"
    />
  </div>
</DocumentFragment>
`;
